autoload/go/lint_test.vim

```diff
@@ -89,11 +91,11 @@ func! Test_GometaAutoSave() abort
 
   call go#lint#Gometa(1)
 
-  let actual = getqflist()
+  let actual = getloclist(l:winnr)
   let start = reltime()
   while len(actual) == 0 && reltimefloat(reltime(start)) < 10
     sleep 100m
-    let actual = getqflist()
+    let actual = getloclist(l:winnr)
```

Comment:
These commands should be not removed, instead it should be based on `go#list#Type("GoMetaLinterAutoSave")` and act accordingly, whether it's  a location list or quickfix list.

Reply:
Same comment as above.